src/isPalindrome.js

```diff
@@ -20,6 +20,11 @@
  */
 function isPalindrome(str) {
   // write code here
+  let lineOfWords = str.toLowerCase().replace(/[^\w]/g, '');
```

Comment:
suggestion: stringOfWords or wordsStr/wordsString

line is not typical name for something that is string.